lib/membrane_caps_mp4/mp4.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Membrane.MP4 do
+  @moduledoc """
+  This module provides caps struct for MP4 container buffers.
```

Comment:
These are not caps, this is an MP4 format if anything 😄 
```suggestion
  This module provides format definition for MP4.
```